docs/tls.md

```diff
@@ -26,7 +26,7 @@ This approach involves acquiring a CA signed or self-signed certificate and use
 kubectl create secret generic tls-ssl-minio --from-file=path/to/private.key --from-file=path/to/public.crt
 ```
 
-Once created, set the name of Secret (here it is `tls-ssl-minio`) under `spec.externalCertSecret` field. Then create the Tenant. MinIO Operator will use this Secret to fetch key and certificate and mount it to relevant locations inside the Tenant pods. 
+Once created, set the name of Secret (here it is `tls-ssl-minio`) under `spec.externalCertSecret` field. Then create the Tenant. MinIO Operator will use this Secret to fetch key and certificate and mount it to relevant locations inside the Tenant pods. Make sure you explicitly set `requestAutoCert` field to `false` to disable automatic generation.
```

Comment:
This is not necessary however the wording should be made optional here, I think this document update is not necessary or should be written differently.